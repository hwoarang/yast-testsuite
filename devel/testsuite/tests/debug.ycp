{

include "testsuite.ycp";
include "devel/debug.ycp";

map m = $[
    1 : nil,
    2 : "a",
    3 : `a,
    4 : 1,
    5 : [ 1 ],
    6 : $[ 1 : 2],
];

DUMP("any2tree");
TEST(``(any2items(m)), [], nil);

DUMP("any2items");
TEST(``(any2items(nil)), [], nil);
TEST(``(any2items("a")), [], nil);
TEST(``(any2items(`a)), [], nil);
TEST(``(any2items(1)), [], nil);
TEST(``(any2items([1])), [], nil);
TEST(``(any2items($[1:2])), [], nil);
TEST(``(any2items(m)), [], nil);

}
