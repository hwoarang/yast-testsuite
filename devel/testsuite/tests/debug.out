Dump	any2tree
Return	[`item ("1", ["NIL"]), `item ("2", ["\"a\""]), `item ("3", ["`a"]), `item ("4", ["1"]), `item ("5", [`item ("1", [])]), `item ("6", [`item ("1", ["2"])])]
Dump	any2items
Return	["NIL"]
Return	["\"a\""]
Return	["`a"]
Return	["1"]
Return	[`item ("1", [])]
Return	[`item ("1", ["2"])]
Return	[`item ("1", ["NIL"]), `item ("2", ["\"a\""]), `item ("3", ["`a"]), `item ("4", ["1"]), `item ("5", [`item ("1", [])]), `item ("6", [`item ("1", ["2"])])]
